google-cloud-spanner-executor/src/main/java/com/google/cloud/spanner/executor/CloudClientExecutor.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.google.cloud.executor.spanner;
+package com.google.cloud.spanner.executor;
```

Comment:
Why are we modifying the package?

Reply:
Just to be consistent with the class names prefix we are maintaining in java-spanner repo.
Ex:
1. com.google.cloud.spanner.admin.database.v1
2. com.google.cloud.spanner.admin.instance.v1
3. com.google.cloud.spanner

Reply:
I understand the need to be consistent but not at the cost of introducing a breaking change. There are other policies a) major version upgrade, b) release with JDK upgrade which needs to be followed if we were to release a breaking change in Java. I think in this case, lets continue to have the same package structure. The lack of consistency is not a major problem (especially when I think of a breaking change).

Reply:
Removed the changes as discussed.